### PR TITLE
Acceptance tests

### PR DESCRIPTION
**Pull request by EXC3ll3NTrhyTHM**
The main updates are in the acceptanceTests folder. I did rename some variables and functions and changed how the Turn and CellType is returned in certain functions. That will conflict with Suryansh's updates and I will need resolve those. So he should merge his changes first because I branched off of his branch.

Almost all tests are passing currently, I'll fix them if Suryansh doesnt see the same failing tests:
image

### CODE REVIEW

File: backend/GameManager.py [hunk 1]
```diff
@@ -17,37 +19,44 @@ def get_board(self, row=-1, col=-1):
...
         if (row, column) in self.current_moves:
-            if self.board.check_position(row, column) != "empty":
-                print(self.board.check_position(row, column).lower())
+            if self.board.check_position(row, column) != CellType.EMPTY:
+                print(self.board.check_position(row, column))
                 return "GameManagerError -- position not empty"
-            
-            flag = 0
+
+            isPiecePlaced = 0
```
Comment (raspberrymilkyway) [inline backend/GameManager.py:36]: This variable shouldn't be camelCase - rename is_piece_placed

File: frontend/BoardGUI.py [hunk 2]
```diff
@@ -1,20 +1,29 @@
 import pygame as pg
-# from backend.Board import Board
+
+from backend.Board import Board
```
Comment (raspberrymilkyway) [inline frontend/BoardGUI.py:3]: I maintain that we should not be importing Board here (separation of frontend and backend; Board gets initialized but never used outside of tests), but that's not something that needs to be fixed for this sprint.

File: frontend/GameGUI.py [hunk 3]
```diff
@@ -1,20 +1,27 @@
-import pygame as pg
 import sys
-from frontend.BoardGUI import BoardGUI 
+
+import pygame as pg
```
Comment (raspberrymilkyway) [inline frontend/GameGUI.py:3]: Out of curiosity, why did the sys and pygame imports get swapped? Was it just to group the various imports better?

File: backend/Board.py [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
 from backend.Cell import Cell, CellType
 
+
 class Board:
     def __init__(self, size):
         self.board_size = size
         self.create_initial_board(size)
-        
-                       
+
```
Comment (raspberrymilkyway) [inline backend/Board.py:9]: Curious -- are these empty line changes from the auto-formatters?

File: frontend/BoardGUI.py [hunk 5]
```diff
@@ -1,20 +1,29 @@
...
 from backend.GameManager import GameManager
-# import backend.Cell as CellType
+from backend.Piece import Turn
```
Comment (raspberrymilkyway) [inline frontend/BoardGUI.py:5]: Hmm, maybe Turn should have its own file then, just to better separate everything...

**Review by raspberrymilkyway — APPROVED** [review]
The only real change that I think should absolutely be fixed (outside of any merge conflicts with Suryansh) is the camelCase variable name -- everything else looks good. Thank you for tackling the AC tests!

File: frontend/BoardGUI.py (hunk 2 above)
Comment (EXC3ll3NTrhyTHM) [reply frontend/BoardGUI.py:3]: Lets update this next spring. I agree it should be built from the GameManager I believe

File: backend/Board.py (hunk 4 above)
Comment (EXC3ll3NTrhyTHM) [reply backend/Board.py:9]: Yeah thats correct. We can adjust it but the AI configured it to match supposedly the Microsoft formatting.

File: frontend/BoardGUI.py (hunk 5 above)
Comment (EXC3ll3NTrhyTHM) [reply frontend/BoardGUI.py:5]: Agreed

File: frontend/BoardGUI.py (hunk 2 above)
Comment (EXC3ll3NTrhyTHM) [reply frontend/BoardGUI.py:3]: https://github.com/Advanced-Software-Engineering-UMKC/Nine-Mens-Morris/issues/32

File: frontend/BoardGUI.py (hunk 5 above)
Comment (EXC3ll3NTrhyTHM) [reply frontend/BoardGUI.py:5]: https://github.com/Advanced-Software-Engineering-UMKC/Nine-Mens-Morris/issues/32

File: frontend/BoardGUI.py (hunk 2 above)
Comment (EXC3ll3NTrhyTHM) [reply frontend/BoardGUI.py:3]: https://github.com/Advanced-Software-Engineering-UMKC/Nine-Mens-Morris/issues/33

File: frontend/GameGUI.py (hunk 3 above)
Comment (EXC3ll3NTrhyTHM) [reply frontend/GameGUI.py:3]: Not sure. It couldnt have been alphbetical, maybe `sys` is directly identified as needed to be in top position.

File: tests/backend/test_Board.py [hunk 6]
```diff
@@ -13,50 +15,46 @@ def board():
...
 
     def test_set_position(self, board):
         # Test if set_position updates to desired state
         board.set_position(0, 0, "white")
         state = board.check_position(0, 0)
-        assert state == "white"
-
+        assert state == CellType.WHITE
 
     def test_create_board_indices_array(self, board):
```
[the remark this answers is not in the document]
Comment (EXC3ll3NTrhyTHM) [reply tests/backend/test_Board.py]: Updated in this commit 9c90828effbc08e240fd54975240748b60c7011d

File: backend/GameManager.py (hunk 1 above)
Comment (EXC3ll3NTrhyTHM) [reply backend/GameManager.py]: Updated in this commit 9c90828effbc08e240fd54975240748b60c7011d

**Pull request merged**